webservices/resources/elections.py

```diff
@@ -60,7 +60,7 @@ def get(self, **kwargs):
 
     def _get_elections(self, kwargs):
         """Get elections from ElectionsList model."""
-        query = db.session.query(ElectionsList).order_by(ElectionsList.district)
+        query = db.session.query(ElectionsList).order_by(ElectionsList.sort_order, ElectionsList.district)
```

Comment:
I think we want to add this as the default sort order, so that people can over-ride it fi they want to. You should be able to add a default sort here https://github.com/18F/openFEC/pull/2864/commits/5a39eb3e398c9e8f081e8d0d898e5db370575b44#diff-1fb1f7e2bd8c60cb7128c0c501821eeaR55